datasets/historical/eia_861_annual_energy_use_state_sector/dataset.json5

```diff
@@ -25,13 +25,12 @@
       pivoted_dimension_type: "metric",
     },
     data_schema_type: "one_table",
-  },
+  },      
   dimensions: [
     {
-      // this is different than the project supplemental state dimension (it includes AK and HI)
       class: 'State',
       description: 'US States',
-      file: 'dimensions/states.csv',
+      file: 'dimensions/states.csv',  // this is different than the project supplemental state dimension (it includes AK and HI)
```

Comment:
The supplemental state dimension could be "conus-state" or "state-conus" instead of just "state".